run-test

```diff
@@ -1,13 +1,35 @@
 #!/bin/bash -e
+# For coverage tests, run as
+#     coverage erase
+#     PATH=$PWD:$PATH RUNNER="coverage run --include=$PWD/**/*" ./run-test
+#     coverage html
+# See ./run-coverage
```

Comment:
It's just an explanation on why you would want to set `RUNNER` variable in the environment. You also might want to use a different python version, e.g. `RUNNER=python3 ./run-test`.